src/tics.jl

```diff
@@ -0,0 +1,114 @@
+function _tics{T<:AbstractFloat}(radec_min::T, radec_max::T, numx::T, ticsize::T, ra::Bool)
+
+    numtics = numx/ticsize
+    if ra
+        mul = 4.0
```

Comment:
Here, and in the other assignments below, you can write `T(4)` instead of `4.0`.